static/img/404-dark.svg

```diff
@@ -0,0 +1,21 @@
+<svg width="200" height="200" viewBox="0 0 200 200" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
In my opinion it's a bad practice to hold 2 identical images
maybe we can manage it something like this?
https://www.ctrl.blog/entry/svg-embed-dark-mode.html

Reply:
Yes, I agree, I've though about something like `fill` manage, but this picture uses at least 3 different colors